### ISSUE TITLE
Repeated notices

### ISSUE THREAD

**Issue by joiglifberg** [ashhitch/wp-graphql-yoast-seo#118]
We get these notices in the logs very frequently.

```
PHP Notice:  Trying to get property 'user_nicename' of non-object in /site/web/app/mu-plugins/wp-graphql-yoast-seo/wp-graphql-yoast-seo.php on line 714
PHP Notice:  Undefined index: bctitle-archive-wpseo in /site/web/app/mu-plugins/wp-graphql-yoast-seo/wp-graphql-yoast-seo.php on line 192
```

**Comment by ashhitch** [comment]
Thank will take a look at this

**Comment by ashhitch** [comment]
This should now be resolved in v4.16.1

**Comment by rodrigo-arias** [comment]
@ashhitch I still can see a similar issue in a different line of code. I created this PR https://github.com/ashhitch/wp-graphql-yoast-seo/pull/130 to fix that one.

```
PHP Notice:  Trying to get property 'user_nicename' of non-object in .../app/public/wp-content/plugins/headless/vendor/ashhitch/wp-graphql-yoast-seo/wp-graphql-yoast-seo.php on line 743
```
